shim/shim.c

```diff
@@ -368,7 +368,7 @@ read_IO_message(struct cc_shim *shim, uint64_t *seq, ssize_t *stream_len) {
 			}
 		}
 	}
-	*seq = get_big_endian_64(buf);
+	*seq = get_big_endian_64((uint8_t*)buf);
```

Comment:
I have used string functions at some other places. I would have to cast to char* there.